packages/react-components/src/components/Drawer/DrawerMenu.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { merge, Stack } from '@fluentui/react';
+import React, { useMemo } from 'react';
+import { _DrawerSurfaceProps } from '.';
+import { BaseCustomStyles } from '../../types';
+import { DrawerMenuItem, _DrawerMenuItemProps } from './DrawerMenuItem';
+import { _DrawerSurface, _DrawerSurfaceStyles } from './DrawerSurface';
+
+/**
+ * Styles for the {@link _DrawerMenu}.
+ *
+ * @internal
+ */
+export interface _DrawerMenuStyles extends BaseCustomStyles {
+  /** Styles for the {@link DrawerSurface} container. */
+  drawerSurfaceStyles?: _DrawerSurfaceStyles;
+}
+
+/**
+ * Props for the {@link _DrawerMenu}
+ *
+ * @internal
+ */
+export interface _DrawerMenuProps {
+  items: _DrawerMenuItemProps[];
+
+  /**
+   * Callback when the drawer's light-dismissal is triggered.
```

Comment:
From a quick search, it seems to me that "light dismisal" is a Microsoft UI surfaces specific term.

https://docs.microsoft.com/en-us/windows/apps/design/controls/dialogs-and-flyouts/flyouts

Should we define / or link to a more "standard" reference to understand what this means / when it is triggered.

Reply:
MUI just has onClose and doesn't refernece the clickableOutside nature: https://mui.com/api/drawer/ (but does have a "click-away listener https://mui.com/components/click-away-listener/)

Reply:
As this is internal only I will leave as lightDismiss as this is inline with fluent's nomenclature. We can revisit the API naming if we decide to make this a publicly available component